### DISCUSSION TITLE
Jira Service Manager

### DISCUSSION THREAD

**Discussion by Mariamyoussef192** [community/community#56985] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello 👋
I'm Mariam Youssef
I'm so happy to join with you and I hope to all My best regards ❤️❤️

**Comment by tuves** [comment]
👋 Welcome to the GitHub Community on Discussions, @Mariamyoussef192! Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
